<!DOCTYPE HTML>
<!--
	Astral by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
<head>
	<title>Nataša Puzović</title>
	<meta charset="utf-8" />
	<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
	<link rel="stylesheet" href="assets/css/main.css" />
	<link rel="icon" href="images/kiwi-bird-solid.svg">
	<noscript><link rel="stylesheet" href="assets/css/noscript.css" /></noscript>
</head>

		<!-- Wrapper-->
		<div id="wrapper">

			<center>
				&nbsp;&nbsp;&nbsp;
				<br>
				<img src="images/myphoto.png" class="centerImage" width="200" height="200" alt="" />
				</center>
				<br>
				<center>
				<h1><font color="white">Nataša Puzović, PhD</font></h2>
				</center>
				&nbsp;
			
				<center>
				<a class="five" href="https://scholar.google.com/citations?user=bfdwYG4AAAAJ&hl=en&oi=ao"
				title="Google Scholar profile"
				target="_blank"><img src="images/googlescholar_logo.png" width="30" height="30"></a>
				
				&thinsp;&thinsp;
			
				<a class="five" href="https://orcid.org/0000-0003-0969-6115"
				title="Orcid profile"
				target="_blank"><img src="images/ORCID-iD_icon_reversed_vector.svg" width="30" height="30"></a>
				
				&thinsp;&thinsp;
			
				<a class="five" href="https://www.researchgate.net/profile/Natasa-Puzovic"
				title="Research Gate profile"
				target="_blank"><img src="images/researchgate.png" width="30" height="30"></a>
			
				&thinsp;&thinsp;
			
				<a class="five" href="https://github.com/NatashaPuzovic"
				title="Github profile"
				target="_blank"><img src="images/github-logo.png" width="30" height="30"></a>
				
				&thinsp;&thinsp;
			
				<a class="five" href="https://twitter.com/natashapuzovic"
				title="Twitter profile"
				target="_blank"><img src="images/twitter-logo.png" width="30" height="30"></a>
				
				<br>
				
			
					<!--<a class="icon solid fa-briefcase fa-lg" style="color:#ffffff"></a>
					<font color="white"> PhD Student </font>
					<br>-->
			
					<!--<a class="icon solid fa-map-marker-alt fa-lg" style="color:#ffffff"></a> 
					<a class="five" href="https://www.evolbio.mpg.de/2169/en" target="_blank">
					Max Planck Institute for Evolutionary Biology </a>
					<br>-->
					
					<a class="icon solid fa-envelope fa-lg" style="color:#ffffff"></a> &thinsp;&thinsp;
					<font color="white"> puzovic [at] evolbio.mpg.de &thinsp; | &thinsp; npuzovic [at] protonmail.com</font>
			
				</center>

				<br>
			<!-- Navigation bar on top -->
			<nav id="nav">
				<a href="#" class="icon solid fa-home"><span>Home</span></a>
				<a href="#research" class="icon solid fa-microscope"><span>Research</span></a>
				<a href="#sideline" class="icon solid fa-directions"><span>Sideline</span></a>
				<!--<a href="#conferences" class="icon solid fa-users"><span>Conferences</span></a> -->
				<!--<a href="#drawings" class="icon solid fa-palette"><span>Drawings</span></a> -->
				<a href="#cv" class="icon solid fa-scroll"><span>CV</span></a>
				<!-- <a href="#outreach" class="icon solid fa-bullhorn"><span>Outreach</span></a> -->
			</nav>
<!------------------------------------------------------------------------------------------------------------------------------------------------------>


<!-- Main -->
				<div id="main">
					<!-- Home tab ----------------------------------------------------------------------------------------------->
						<article id="home" class="panel intro">
							<header>								
								<h2 style="color:rgb(133, 133, 133);"> About me </h2> 
								<br>
								I'm a biologist interested in evolution, development, systems biology and complex systems.
								I'm trained in both wetlab and drylab biological methods, but nowadays I prefer the computer desk to the lab bench. 
								My main interests are studying gene expression and molecular networks in the cell, and their evolution. 
								<br><br>
								What particularly tickles my fancy:
								<ul>
									<li>gene expression level evolution, i.e. how gene expression level changes over time and through which regulatory mechanisms.</li>
									<li>cell differentiation, i.e. how changes in gene expression lead cells from one phenotypic state to another, or to novel phenotypes.</li>
									<li>interactions between genes (gene networks) and interactions between cells and tissues in developing embryos.</li>
									<li>computational or mathematical models of gene expression/gene networks/cells, especially dynamical models (which incorporate the time dimension), and testing their predictions with experimental data.</li>
									<li>simulations of the evolutionary process by rounds of reproduction, selection and mutation (in silico experimental evolution).</li>
									<li>non-reductionist approaches in biological research, i.e. looking at organisms as systems whose parts interact with one another instead of looking at organisms as independent sets of SNPs or genes.</li>
									<li>discovery-driven science, instead of sloppy science driven by chasing the next minimal publishable unit.</li>
								  </ul>
							</header>
							<a href="#research" class="jumplink pic">
								<span class="arrow icon solid fa-chevron-right"><span>See my work</span></span>
								<img src="images/codex.png" alt="" />
							</a>
						</article>

					<!-- Research tab ----------------------------------------------------------------------------------------------->
						<article id="research" class="panel">

							<h3> Evolution of gene expression in gene regulatory networks in changing environments</h3>
							<br>
							The variability of gene expression levels, also known as gene expression noise, is an evolvable trait subject to selection. While gene expression noise is detrimental in constant environments where the expression level is under stabilizing selection, it may be beneficial in changing environments when the phenotype is far from the optimum. However, expression noise propagates along the gene network, making the evolution of connected genes interdependent. I explored how their position in the gene network constrains the evolution of genes under selection using an in silico evolution experiment. I simulated the evolution of populations of model gene regulatory networks under directional and fluctuating selection on the gene expression level while allowing the basal expression level and expression noise level to mutate. I found that expression noise is only transiently favoured under directional selection, but high levels of noise can be maintained in a fluctuating selection regime. Furthermore, target genes, regulated by other genes, were more likely to increase their gene-specific expression noise than regulator genes. These findings suggest that both the mean and variance of gene expression levels respond to selection due to changing environments – and do so in a network-dependent manner. They further point at gene expression noise as a putative mechanism for populations to escape extinction when facing environmental changes.
							<br>
							<br>
							<center>
							<img src="images/plot_m2_fig.png" class="centerImage" height="250" alt="" />
							</center>
							<br>
							<i>Published as article preprint titled "When is gene expression noise advantageous?" on bioRxiv.
								<br> Read article <a href="https://doi.org/10.1101/2023.12.04.569843" target="_blank">here</a> 
								or download PDF <a href="https://www.biorxiv.org/content/10.1101/2023.12.04.569843v1.full.pdf" target="_blank">here</a>.</i>
							<br><br>
							Reading is boring? Watch a 10min video summary instead:
							<center>
								<iframe width="560" height="315" src="https://www.youtube.com/embed/CxrTF_kmkRE?si=TUYQY_AcH39gAild" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
							</center>
							
							<hr>					
							<br>
							
							<h3>Evolution of gene expression noise in gene regulatory networks</h3>
							<br>
							Expression noise, the variability of the amount of gene product among isogenic cells grown in identical conditions, originates from the inherent stochasticity of diffusion and binding of the molecular players involved in transcription and translation. It has been shown that expression noise is an evolvable trait and that central genes exhibit less noise than peripheral genes in gene networks. A possible explanation for this pattern is increased selective pressure on central genes since they propagate their noise to downstream targets, leading to noise amplification. To test this hypothesis, I developed a new gene regulatory network model with inheritable stochastic gene expression and simulated the evolution of gene-specific expression noise under constraint at the network level. I observed that local network features affect both the probability to respond to selection, and the strength of the selective pressure acting on individual genes. In particular, the reduction of gene-specific expression noise as a response to stabilizing selection on the gene expression level is higher in genes with higher centrality metrics. Furthermore, global topological structures such as network diameter, centralization and average degree affect the average expression variance and average selective pressure acting on constituent genes. These results demonstrate that selection at the network level leads to differential selective pressure at the gene level, and local and global network characteristics are an essential component of gene-specific expression noise evolution.
							<br><br>
							<center>
							<img src="images/journal.ploscompbiol.2023.F6.PNG" class="centerImage" height="280" alt="" />
							</center>
							<br>
							<i>Published as article titled "Being noisy in a crowd: Differential selective pressure on gene expression noise in model gene regulatory networks" in PLOS Computational Biology.
								<br> Read article <a href="https://doi.org/10.1371/journal.pcbi.1010982" target="_blank">here</a>
								or download open access PDF <a href="pdfs/journal.ploscompbiol.2023.pdf" target="_blank">here</a>.</i>
							<br><br>
							Reading is boring? Watch a 10min video summary instead:
							<center>
								<iframe width="560" height="315" src="https://www.youtube.com/embed/svjoNXo-OCc?si=EpwMTteM1Gcwkazj&amp;start=903" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>							
							</center>
							
							<br><br>

							
						</article>


					<!-- sideline tab ----------------------------------------------------------------------------------------------->
						<article id="sideline" class="panel">
						<h3>Complex systems - the concept of resilience in biology</h3>
						<br>
						During my PhD I had the opportunity to attend a complex systems summer school organized by the Santa Fe Institute.
						I was interested in the ability of biological systems to undergo internal perturbations like noise and maintain their function, so I joined a working group which set out to study the concept of resilience as a project. 
						We looked into how different disciplines understand resilience and compared them in an attempt to find the essential aspects and provide a unified, interdisciplinary definition.
						<br><br>					
						<center>
							<img src="images/plot_frontiers_fig.jpg" class="centerImage" height="200" alt="" />
						</center>
						<br>
						<i>Published as article titled "Resilience - Towards an interdisciplinary definition using information theory" in Frontiers in Complex Systems.
							 Read article <a href="https://doi.org/10.3389/fcpxs.2023.1236406" target="_blank">here</a>.</i>
						
						<hr>	
						<br>

						<h3>Philosophy of biology - is an organism a machine?</h3>
						<br>
						In my PhD I studied how randomness in gene expression level can be selectively advantageous or disadvantageous depending on the environmental conditions.
						Randomness/stochasticity is often overlooked when thinking about molecular mechanisms and in the discussion of my thesis I wrote about why that is not only incorrect, but can also be harmful to our understanding of living beings.
						<br><br>
						<center>
							<img src="images/figDiscussion.png" class="centerImage" height="350" alt="" />
						</center>
						<br>
						<i>Read it <a href="https://macau.uni-kiel.de/servlets/MCRFileNodeServlet/macau_derivate_00005362/Doctoral_thesis.pdf" target="_blank">here</a>.</i>
						</article>


					<!-- Conferences tab ----------------------------------------------------------------------------------------------->
					<!--	<article id="conferences" class="panel">

							<!-- Talks subsection 
							<header>
								<h2> Talks </h2>
							</header>
									<h3> International Max Planck Research School for Evolutionary Biology Retreat 2021 </h3>
										Oral presentation
										<br>
										"Noisy neighbours will be reported - how gene network topology
										affects the evolution of gene-specific expression noise"
										<br>
										Kiel, Germany, 16-17 Sept 2021 
	
							<br><br>
									<h3> Aquavit 2021 </h3>
										Oral presentation
										<br>
										"The impact of gene network topology on the evolution of gene-specific expression noise"
										<br>
										Virtual, 17-18 June 2021 

							<br><br>
									<h3> Interdisciplinary Approach to Molecular Evolution 2020 </h3>
									<h3>Approche Interdisciplinaire de l'Evolution Moléculaire (AIEM)</h3>
										Oral presentation
										<br>
										"The impact of gene network topology on the evolution of gene-specific expression noise"
										<br>
										Virtual, 15-17 December 2020 
							<br><br>

									<h3> Society for Molecular Biology & Evolution Annual Meeting 2019 </h3>
										Oral presentation
										<br>
										"The impact of gene network topology on the evolution of gene-specific expression noise"
										<br>
										Session: Evolution of Proteins and Molecular Pathways/Networks under Higher Level Selective Effects
										<br>
										Manchester, United Kingdom 21-15 July 2019 
							<br><br>

							
							<!-- Poster presentations subsection 
							<header>
								<h2> Poster presentations </h2>
							</header>
										<h3> International Conference of Systems Biology (ICSB) 2022 </h3>
										Poster presentation
										<br>
										"Being noisy in a crowd: differential selective pressure on gene expression noise in model gene regulatory networks"
										<br>
										Berlin, Germany 08-12 October 2022
										<br>
										Download poster
										<a class="fas fa-file-pdf" style="font-size:24px;color:rgb(255, 137, 137)" href="pdfs/poster_ICSB2022.pdf" target="blank"></a>
							<br><br>
									<h3> Society for Molecular Biology & Evolution Annual Meeting 2021 </h3>
										Poster presentation
										<br>
										"The impact of gene network topology on the evolution of gene-specific expression noise"
										<br>
										Virtual, 03-08 July 2021
							<br><br>

									<h3> Cold Spring Harbor Laboratory Probabilistic Models in Genomics 2021 </h3>
										Poster presentation
										<br>
										"The impact of gene network topology on the evolution of gene-specific expression noise"
										<br>
										Virtual, 14-16 April 2021
							<br><br>

									<h3> Cold Spring Harbor Laboratory Network Biology Meeting 2021 </h3>
										Poster presentation
										<br>
										"The impact of gene network topology on the evolution of gene-specific expression noise"
										<br>
										Virtual, 16-19 March 2021 
							<br><br>

									<h3> Aquavit 2020 </h3>
										Poster presentation
										<br>
										"The impact of gene network topology on the evolution of gene-specific expression noise"
										<br>
										Virtual, 15-17 December 2020
							<br>
							
						</article> -->


						<!-- CV tab ----------------------------------------------------------------------------------------------->
						<article id="cv" class="panel">
							<h3>Download CV
							<a class="fas fa-file-pdf" style="font-size:24px;color:rgb(255, 137, 137)" href="pdfs/CV.pdf" target="blank"></a>
							</h3>
							<hr>

							<h2> Education</h2>
								<ul>
									<li>PhD Evolutionary Biology (magna cum laude), Christian-Albrechts-Universität zu Kiel, Germany (2020-2023)</li>
									<li>MSc Molecular Biology and Evolution, Christian-Albrechts-Universität zu Kiel, Germany (2017-2019)</li>
									<li>BSc Molecular Biology and Physiology, Faculty of Biology, University of Belgrade, Serbia (2012-2017)</li>
									<li>BSc Applied Mathematics (unfinished), Faculty of Mathematics, University of Belgrade, Serbia (2014-2016)</li>
								</ul>
															
							<h2> Publications </h2>
								<li><b>Puzović, N.</b> & Dutheil, J. Y. (2023). When is gene expression noise advantageous? bioRhiv Preprint.
									<br>&nbsp;&nbsp;&nbsp;&nbsp; <a href="https://doi.org/10.1101/2023.12.04.569843" target="_blank">https://doi.org/10.1101/2023.12.04.569843</a> </li>
								<li><b>Puzović, N.</b>, Madaan, T. & Dutheil, J. Y. (2023). Being noisy in a crowd: Differential selective pressure on
									gene expression noise in model gene regulatory networks. PLOS Computational Biology.
									<br>&nbsp;&nbsp;&nbsp;&nbsp; <a href="https://doi.org/10.1101/2022.08.01.502352" target="_blank">https://doi.org/10.1101/2022.08.01.502352</a> </li>
								<li>Nisioti, E., Clark, C., Kunal Das, K., Ekkehard, E., Friedenberg, N. A., Gates, E., Lambros, M., Lazurko, A.,
									<b>Puzović, N. </b> & Salas, I. (2023). Resilience—towards an interdisciplinary definition using information theory.
									Frontiers in Complex Systems.
									<br>&nbsp;&nbsp;&nbsp;&nbsp; <a href="https://doi.org/10.3389/fcpxs.2023.1236406" target="_blank">https://doi.org/10.3389/fcpxs.2023.1236406</a> </li>
								<li>V. Barroso, G., <b>Puzović, N.</b> & Dutheil, J. Y. (2019). Inference of recombination maps from a single pair of
									genomes and its application to ancient samples. PLOS Genetics, 15(11), 1–21.
									<br>&nbsp;&nbsp;&nbsp;&nbsp; <a href="https://doi.org/10.1371/journal.pgen.1008449" target="_blank">https://doi.org/10.1371/journal.pgen.1008449</a> </li>
								<li>Barroso, G. V., <b>Puzovic, N.</b> & Dutheil, J. Y. (2018). The evolution of gene-specific transcriptional noise is
									driven by selection at the pathway level. Genetics, 208(1), 173–189.
									<br>&nbsp;&nbsp;&nbsp;&nbsp; <a href="https://doi.org/10.1534/genetics.117.300467" target="_blank">https://doi.org/10.1534/genetics.117.300467</a></li>
							<br>

							<h2> Research Experience </h2>
								<li>Molecular Systems Evolution Research Group, PhD researcher <br>
								&nbsp;&nbsp;&nbsp;&nbsp; Department of Theoretical Biology, Max Planck Institute for Evolutionary Biology, Plön (2020-2023)</li>
								<li>Genomic Microbiology Group, PhD rotation <br>
								&nbsp;&nbsp;&nbsp;&nbsp; Institute of General Microbiology, Kiel University, Kiel (2019)</li>
								<li>Stochastic Evolutionary Dynamics Research Group, PhD rotation <br>
								&nbsp;&nbsp;&nbsp;&nbsp; Department of Evolutionary Theory, Max Planck Institute for Evolutionary Biology, Plön (2019)</li>
								<li>Molecular Systems Evolution Research Group, Master’s thesis project <br>
								&nbsp;&nbsp;&nbsp;&nbsp; Department of Evolutionary Genetics, Max Planck Institute for Evolutionary Biology, Plön (2019)</li>
								<li>Laboratoire Matière et Systèmes Complexes (Laboratory for Matter and Complex Systems), Internship <br>
								&nbsp;&nbsp;&nbsp;&nbsp; UMR 7057 CNRS, Université Paris 7 Diderot, Université Sorbonne Paris Cité (2018 – 2019)</li>
								<li>Tautz Research Group, Internship <br>
								&nbsp;&nbsp;&nbsp;&nbsp; Department of Evolutionary Genetics, Max Planck Institute for Evolutionary Biology, Plön (2018)</li>
								<li>Molecular Systems Evolution Research Group, Research Assistant <br>
								&nbsp;&nbsp;&nbsp;&nbsp; Department of Evolutionary Genetics, Max Planck Institute for Evolutionary Biology, Plön (2017 – 2018)</li>
								<li>Molecular Systems Evolution Research Group, Internship <br>
								&nbsp;&nbsp;&nbsp;&nbsp; Department of Evolutionary Genetics, Max Planck Institute for Evolutionary Biology, Plön (2015, 2016)</li>
							<br>	

							<h2> Awards and Honors </h2>							
								<li> Magna cum laude honors for the PhD dissertation, graded 1.0/1.0 (scale 1.0-4.0, best 1.0),
									awarded by Christian-Albrechts-Universität zu Kiel, Germany (2023)</li>
								<li>International Max Planck Research School for Evolutionary biology PhD grant,
									granted by the Max Planck Society and the Max Planck Institute for Evolutionary Biology (2019)</li>
								<li>ERASMUS+ Internship Scholarship, granted by ERASMUS+ of European Union (2018)</li>
								<li>Fellowship of the City of Belgrade, granted by the City Administation of Belgrade (2010, 2011)</li>
							<br>

							<h2> Conferences (selected)</h2>							
								<li> International Max Planck Research School for Evolutionary Biology Retreat, oral presentation
									<br> &nbsp;&nbsp;&nbsp;&nbsp; Kiel, Germany, 16-17 Sep 2021 </li>
								<li> Society for Molecular Biology & Evolution Annual Meeting (SMBE), virtual poster presentation
									<br> &nbsp;&nbsp;&nbsp;&nbsp; Society for Molecular Biology & Evolution, 03-08 Jul 2021 </li>
								<li> Aquavit 2021, virtual poster presentation
									<br>  &nbsp;&nbsp;&nbsp;&nbsp; Max Planck Institute for Evolutionary Biology, Plön, Germany, 17-18 Jun 2021 </li>
								<li> CSHL Probabilistic Modeling in Genomics, virtual poster presentation
									<br> &nbsp;&nbsp;&nbsp;&nbsp; Cold Spring Harbor Laboratory, New York, USA, 14-16 Apr 2021 </li>
								<li> CSHL Network Biology Meeting, virtual poster presentation
									<br> &nbsp;&nbsp;&nbsp;&nbsp; Cold Spring Harbor Laboratory, New York, USA, 16-19 Mar 2021 </li>
								<li> Interdisciplinary Approach to Molecular Evolution, virtual poster presentation
									<br> &nbsp;&nbsp;&nbsp;&nbsp; CNRS, France, 15-17 Dec 2020 </li>
								<li> Aquavit 2020, virtual poster presentation
									<br> &nbsp;&nbsp;&nbsp;&nbsp; Max Planck Institute for Evolutionary Biology, Plön, Germany, Jun 2020 </li>
								<li> Society for Molecular Biology & Evolution Annual Meeting (SMBE), oral presentation
									<br> &nbsp;&nbsp;&nbsp;&nbsp; Society for Molecular Biology & Evolution, Manchester, United Kingdom, 21-15 Jul 2019 </li>
							<br>

							<h2> Press </h2>
							<li>Being noisy in a crowd: How gene interactions shape the evolution of cell-to-cell variation.
								<br>&nbsp;&nbsp;&nbsp;&nbsp; Max Planck Society Press Release (20. April 2023)
								<br>&nbsp;&nbsp;&nbsp;&nbsp; <a href="https://www.evolbio.mpg.de/3672888/being-noisy-in-a-crowd?c=15503" target="_blank">https://www.evolbio.mpg.de/3672888/being-noisy-in-a-crowd?c=15503</a>
								</li>	
							<br>

							<h2> Supervision Experience </h2>		
								<li> Undergraduate internship of Barbara D'Albis (2022)</li>					
								<li> IMPRS PhD rotation of Nikhil Sharma (2020) </li>
								<li> Undergraduate internship of Tanvi Madaan (2020) </li>
							<br>

							<h2> Skills </h2>
							<li> Computational skills: R, C++, MATLAB, Python  </li>
								<li> Bioinformatic software: MAFFT, iqtree, snakemake </li>
								<li> Other software: Inkscape, Latex </li>
								<li> Spoken languages: English (proficient), German (basic), Serbian-Croatian-Bosnian-Montenegrin (native) </li>

							<br>


						</article>

				</div>
<!------------------------------------------------------------------------------------------------------------------------------------------------------>
					<!-- Footer -->
					<div id="footer">
						<ul class="copyright">
							<li>&copy; Nataša Puzović.</li><li>Design forked from: <a href="http://html5up.net">HTML5 UP</a></li>, edited by NP.
						</ul>
					</div>

					<!-- Scripts -->
					<script src="assets/js/jquery.min.js"></script>
					<script src="assets/js/browser.min.js"></script>
					<script src="assets/js/breakpoints.min.js"></script>
					<script src="assets/js/util.js"></script>
					<script src="assets/js/main.js"></script>
		</div>
	</div>

</div>

</body>
</html>